### PR TITLE
Fix to ConvCorr tests to skip host tests when host not enabled

### PR DESCRIPTION
**Pull request by aayushg55**

### CODE REVIEW

File: test/00_transform/ConvCorr.cu [hunk 1]
```diff
@@ -142,6 +162,19 @@ protected:
   }
 
   void TearDown() override { pb.reset(); }
+
+  void CheckExecSupport() {
```
Comment (cliffburdick) [inline test/00_transform/ConvCorr.cu:166]: This function can be marked constexpr too

File: test/00_transform/ConvCorr.cu [hunk 2]
```diff
@@ -78,6 +79,19 @@ protected:
...
```
Comment (cliffburdick) [inline test/00_transform/ConvCorr.cu:83]: `constexpr` function too

File: test/00_transform/ConvCorr.cu (hunk 1 above)
Comment (aayushg55) [reply test/00_transform/ConvCorr.cu]: Fixed

**Review by cliffburdick — APPROVED** [review]

**Pull request merged**